src/xcode/ENA/ENA/Source/Services/Exposure Transaction/ExposureDetectionExecutor.swift

```diff
@@ -65,7 +65,7 @@ final class ExposureDetectionExecutor: ExposureDetectionDelegate {
 		writtenPackages: WrittenPackages,
 		completion: @escaping (Result<[ENExposureWindow], Error>) -> Void
 	) -> Progress {
-		let progress = Progress(totalUnitCount: 2)
+		let progress = Progress(totalUnitCount: Int64.max)
```

Comment:
Why is this change needed? totalUnitCount will never be reached with this number.

Reply:
I forgot to mention it in the description: There are two assumptions to prevent the crash: on the one side we build in the guards and the other thing is: The crash occurs whenever we want to add another progress, but the max amount is already reached. So why not allow more progresses? I found nothing out, why there are only 2 totalUnitCount allowed, so i took the maximum amount.
Does it give you a stomach ache @ArturFriesen?

Reply:
It looks wrong to me. I think at the moment we are not using the progress unit counts. We only use the progress to cancel it. As soon as we start using the progress unit counts 'Int64.max' will be wrong.